packages/product-react/src/components/Tabs/Tabs.scss

```diff
@@ -0,0 +1,66 @@
+@import '@wework/ray-core/scss/ray-core.scss';
+
+$easing: cubic-bezier(0.694, 0.0482, 0.335, 1);
+$hover-border-size: 2px;
+$hover-animation-time: 0.33s;
+
+@mixin tab-border($border-size, $color) {
+  content: '';
+  position: absolute;
+  top: 100%;
+  height: $border-size;
+  left: 0;
+  width: 100%;
+  background-color: $color;
+}
+
+@mixin underline-hover(
+  $border-size,
+  $transition-timing,
+  $inner-element-bottom-padding: false
+) {
+  position: relative;
+  text-decoration: none;
+
+  &::after {
+    @include tab-border($border-size, $ray-color-black);
+
+    transform-origin: center;
+    transform: scaleX(0);
+    transition: transform $transition-timing $easing; // stylelint-disable-line property-blacklist
```

Comment:
transition here seems much more natural for a hover in/out animation than using keyframes. I have't even found a working solution yet using keyframes to do this... What do we think about this?